lib/chef/provider/systemd_unit.rb

```diff
@@ -58,20 +58,28 @@ def define_resource_requirements
 
       def systemd_unit_status
         @systemd_unit_status ||= begin
+          status = {}
+
           # Collect all the status information for a unit and return it at once
-          # This may fail if we are managing a template unit (e.g. with '@'), in which case
-          # we just ignore the error because unit status is irrelevant in that case
+          # This will fail if we are managing a template unit (e.g. with '@'), which is why
+          # we capture 'is-enabled' below
+          # This fails because '--system show' returns 'enabled' for *all* templated units
+          # if *any* of the templated units are enabled
```

Comment:
This isn't true for modern systemd; it's probably a bug that was fixed at some point along the way.  But I don't know when; 219 was released in 2015.

Reply:
Removed these comments.